Project/Source/Components/UI/ComponentCanvasRenderer.cpp

```diff
@@ -53,3 +53,7 @@ const ComponentCanvas* ComponentCanvasRenderer::AnyParentHasCanvas(GameObject* c
 
 	return nullptr;
 }
+
+bool ComponentCanvasRenderer::CanBeRemoved() const {
+	return GetOwner().GetComponent<ComponentImage>() && GetOwner().GetComponent<ComponentText>() && GetOwner().GetComponent<ComponentToggle>();
```

Comment:
No deberían ser != nullptr?

Reply:
el nullptr equivale a un false en un booleano. Entiendo que devuelve false si el objecto contiene uno de estos componentes. Entonces es correcto

Reply:
Tambien había que poner || en lugar de and, Whoops! Lo testeare en los tres casos para asegurar antes de mergear a develop